test/client-library-secrets.py

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/python
+
+# Python is the common scripting language between the docker images used to
+# test the various Taskcluster client libraries.  This fetches the
+# `testing/client-libraries` secret and writes out shell-compatible export
+# statements for each value it contains.
+
+try:
+    import urllib.request
+    PY27 = False
+except ImportError:
+    import urllib
+    PY27 = True
```

Comment:
What is the reason to support python 2.7?

Reply:
The TC client still supports it, and also that's the version of Python available in the node images.  I wanted to write one script to do this secret-fetching that would work in all of the test docker images, and py27 was the common factor.